Update windows instructions (#310)
NUbots/NUbook: src/book/03-guides/01-main-codebase/01-getting-started.mdx

<Alert type='info'>

> If your computer struggles to build and crashes, do './b build -jx', where x is the number of cores you want to use. A good recommended amount of cores is five, if you still struggle, perhaps use one core.

Reviewer: ```suggestion
If your computer struggles to build and crashes, run `./b build -jx`, where x is the number of cores you want to use. A good recommended amount of cores is five, but if you still struggle perhaps use one core.
```
